scss/navbar.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0px 5px 5px 0px $header-color;
```

Comment:
don't add the `px` when the value is 0. this is because 0 is the same in every unit and it's just for consistency